dropwizard-jsonapi/src/main/java/io/rtr/jsonapi/util/FieldUtil.java

```diff
@@ -0,0 +1,48 @@
+package io.rtr.jsonapi.util;
+
+import com.google.common.collect.Lists;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.List;
+
+public class FieldUtil {
```

Comment:
Unit tests... Might as well start now.  http://sonarqube01.prod.renttherunway.it:9000/drilldown/measures/3805?metric=coverage